### PR TITLE
Upgrade react dropzone to v11.1.0

### LINKED ISSUE #1682: Update react-dropzone to newer version
Right now react-dropzone is pinned at a very old 5.1.0 version, while the latest is 11.something. It wouldn't be a problem in itself, except that the API is very different and Volto developers need to be aware of this fact, to research older documentation.

### PR DESCRIPTION
**Pull request by nileshgulia1**
We're using quite old react-dropzone package. The latest being 11.1.0 and we were still at v5.x. closes #1682

### CODE REVIEW

File: src/components/manage/Contents/ContentsUploadModal.jsx [hunk 1]
```diff
@@ -252,7 +262,12 @@ class ContentsUploadModal extends Component {
                       </Table.Cell>
                       <Table.Cell>
                         {file.type.split('/')[0] === 'image' && (
-                          <Image src={file.preview} height={60} />
+                          <Image
+                            src={`${flattenToAppURL(
```
Comment (nileshgulia1) [inline src/components/manage/Contents/ContentsUploadModal.jsx:266]: @tiberiuichim Latest react-dropzone package hasn't has `preview` property. How do we internally handle image previews when images are coming not from `@image` endpoints?

Comment (nileshgulia1) [reply src/components/manage/Contents/ContentsUploadModal.jsx:269]: https://github.com/plone/volto/blob/88b6ea08580436f7a3614f07cab52711238acd97/src/components/manage/Widgets/FileWidget.jsx#L66

Comment (nileshgulia1) [reply src/components/manage/Contents/ContentsUploadModal.jsx]: 4bee4f2 fixes it.

**Review by tiberiuichim — CHANGES_REQUESTED** [review]
There's an error in the folder contents multi-upload, it happens when uploading multiple images. Only one of them is properly previewed, when I try to remove the "broken" ones I get this error:

```
(TypeError): Cannot set property 'src' of null
(anonymous function)
src/components/manage/Contents/ContentsUploadModal.jsx:128
  125 | getUrl = (file) => {
  126 |   readAsDataURL(file).then((data) => {
  127 |     const fields = data.match(/^data:(.*);(.*),(.*)$/);
> 128 |     this.previewRef.current.src = fields[0];
      | ^  129 |   });
  130 | };
  131 | /**
```

**Review by iFlameing — APPROVED** [review]

**Comment by iFlameing** [comment]
@tiberiuichim I fixed the problem which you mentioned. Can you please review it once again??

**Comment by tisto** [comment]
@tiberiuichim can you do another review of this PR? @iFlameing would like to work on the upload drop zone highlighting feature which depends on this PR to be merged.

**Review by nileshgulia1 — COMMENTED** [review]
LGTM. Thanks @iFlameing

**Comment by tiberiuichim** [comment]
@tisto @iFlameing @nileshgulia1  now the upload works as expected, but:

I don't know if it's just my system, I get a console error:

```
React Intl] Error formatting the default message for: "Upload {count, plural, one {# file} other {# files}}"
Error: The intl string context variable "undefined" was not provided to the string "undefined"
```

and the tooltip for the accept button seems to have something broken in its message formatting.
Screenshot 2020-09-25 082430

**Comment by tiberiuichim** [comment]
Screenshot 2020-09-25 192424

**Comment by iFlameing** [comment]
@tiberiuichim I also found the same error in my machine also. But I didn't know why it is giving error I haven't change anything to formating message.

**Comment by iFlameing** [comment]
@tiberiuichim @tisto @nileshgulia1 This error is not introduced by this pr. You can see the same error in master also.

**Review by tiberiuichim — APPROVED** [review]

**Comment by iFlameing** [comment]
@tisto @nileshgulia1 @tiberiuichim I fixed the translation issue. Ready for merge :)

**Comment by iFlameing** [comment]
@tisto It is ready for merge :)

File: locales/de/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -1888,10 +1888,6 @@ msgstr ""
 msgid "Upload files"
 msgstr "Datei hochladen"
 
-#: components/manage/Contents/ContentsUploadModal
-msgid "Upload {count, plural, one {# file} other {# files}}"
-msgstr "Lade {count, plural, one {# Datei} other {# Dateien}} hoch"
```
Comment (tisto) [inline locales/de/LC_MESSAGES/volto.po:1893]: @nileshgulia1 we should keep the German translation here if possible. :)

Comment (tisto) [reply locales/de/LC_MESSAGES/volto.po:1893]: @nileshgulia1 FYI: I re-added the translation myself. Will do a release now.

**Comment by avoinea** [comment]
@tisto Unfortuantely this was a breaking feature https://github.com/plone/volto/issues/1682#issuecomment-661057986. 

Upgrading my votlo app from @plone/volto `8.2.2` to `8.2.3` I get the following error in Edit mode:

2020-10-08-002328_1008x846_scrot

**Comment by tisto** [comment]
@avoinea can you provide steps to reproduce? Does this happen in vanilla Volto? We had to amend some code in our project as well. We might have to consider to revert this then and re-release a new mayor version of Volto.

**Comment by nileshgulia1** [comment]
As this was a major version upgrade of react-dropzone, this will cause a
breaking change in volto projects using older versions of react-dropzone.

**Comment by tisto** [comment]
@nileshgulia1 could you start writing a PR with an update to the Volto upgrade guide describing this?

**Comment by avoinea** [comment]
Sorry, the issue was introduced in `8.2.2`

**Comment by nileshgulia1** [comment]
@tisto Sure, will do.

**Comment by tiberiuichim** [comment]
@tisto It's not a breaking change, it's just something that belongs to our internal "kitchen". We've had a package that depended on react-dropzone, but we haven't specified react-dropzone as dependency. Addon authors, beware. :-)

**Pull request merged**